### ISSUE TITLE
failed when using win64 emacs and cygwin svn

### ISSUE THREAD

**Issue by redguardtoo** [syohex/emacs-git-messenger#29]
I suggest giving user an extra chance to fix the path:

here is the my patch, https://github.com/redguardtoo/emacs.d/commit/ff7480a69cbbf6bd883bd62b05c6209e2d98fc82

here is sample usage from user side:
```
(setq git-messenger:fix-code-file-path-callback
      (lambda (file)
        (message "git-messenger:fix-code-file-path-callback called")
        (setenv "LANG" "")
        (if (string-match "^[cC]:/Workspaces/MAY_2015/" file)
            (setq file (replace-regexp-in-string
                        "^[cC]:/Workspaces/MAY_2015/"
                        "~/projs/lola/" file)))
        file))
```

tested on emacs24.4, windows, cygwin svn

**Comment by syohex** [comment]
`git-messenger` does not use full-path from #31.

**Comment by redguardtoo** [comment]
thanks, you solution is much cleaner.

another suggestion, 
reset env variable before executing svn command and restore it after execution. looks cygwin svn has some issue. see attached screen shot. Git is fine.

![bug](https://cloud.githubusercontent.com/assets/184553/6630918/912f4a82-c972-11e4-9e72-cc3ae5bf1ed0.png)

**Comment by syohex** [comment]
Environment variable `LANG` issue is fixed at #32.
